lib/model/model.ts

```diff
@@ -6158,6 +6158,40 @@ export class CreateUpdatedImageRequest {
     } 
 }
 
+/**
+ * Request model for CreateWebSiteImageFeatures operation.
+ */
+export class CreateWebSiteImageFeaturesRequest {
+    /**
+     * The search context identifier.
+     */
+    public searchContextId: string;
+
+    /**
+     * Images source - a web page
+     */
+    public imagesSource: string;
+
+    /**
+     * Input image
+     */
+    public imageData: Buffer;
```

Comment:
We don`t use parameter imageData: Buffer in the CreateWebSiteImageFeatures method.